mirdata/datasets/simac.py

```diff
@@ -0,0 +1,226 @@
+
+    **Project Overview:**
+
+    SIMAC's approach to music content processing involves the computation of low-level signal features, characterizing the acoustic properties of signals. However, it goes beyond by incorporating higher-level semantic descriptors into these feature sets. These descriptors emphasize the musical attributes of audio signals, like chords, rhythm, and instrumentation, achieving a higher level of semantic complexity than low-level features.
+
+    **Musical Facets and Descriptors:**
+
+    - **Rhythm:** SIMAC investigates various aspects of automatic rhythm description, such as tempo induction, beat tracking, and rhythmic pattern characterization. High-level rhythmic descriptors are used for genre classification of recorded audio, demonstrating the significance of these features in characterizing dance music.
+
+    - **Harmony:** The project explores harmonic aspects of music, defining it through the combination of notes, chords, and their progressions. Harmonic-based retrieval is facilitated without the need for pitch estimation in the mixture, enabling operation on a wide variety of music.
+
+    - **Timbre and Instrumentation:** SIMAC focuses on the overall timbre or texture of music, as current technologies do not allow for reliable separation of individual instrumental information. This aspect is characterized based on low-level signal features.
+
+    - **Music Structure:** The project examines how music materials are presented, repeated, varied, or confronted in a piece, providing ways to interact with audio content through summaries, fast-listening, and on-the-fly identification of songs.
+
+    - **Intensity and Complexity:** These descriptors are defined to capture the subjective sensation of energeticness and the effort required to follow and understand a musical piece. The project notes the relationship between music complexity and listener preference.
+
+    - **Music Similarity:** SIMAC tackles the challenge of defining music similarity, considering both audio-based aspects and cultural background. It aims to develop similarity metrics that incorporate multiple musical facets beyond just timbre.
+
+    **Future Directions and Challenges:**
+
+    SIMAC identifies areas for future exploration, such as incorporating musical facets beyond timbre in similarity metrics and addressing the limitations of current approaches to similarity. The project underscores the potential benefits of enriching music files with metadata from their origin for more effective music retrieval and recommendation systems.
+
+    **Acknowledgments and References:**
+
+    The project involved more than 15 collaborators and was led by teams from Universitat Pompeu Fabra Barcelona, Queen Mary University London, the Austrian Research Institute for Artificial Intelligence Vienna, and Philips Research Eindhoven. For detailed information, visit [http://www.semanticaudio.org](http://www.semanticaudio.org).
+"""
```

Comment:
I am a bit confused by the description... it actually doesn't match the actual data we are loading right? The dataset loader looks like it's a beat, tempo dataset, but this introduction to the dataset presents a broader project. I guess it's the rhythm part? Probably we should stress that a bit more... also in the dataset name and all docs.